community/cypher/compatibility-suite/src/test/scala/cypher/GlueSteps.scala

```diff
@@ -62,40 +66,32 @@ class GlueSteps extends FunSuiteLike with Matchers with ScalaDsl with EN {
     graph.shutdown()
   }
 
+  Background(BACKGROUND) {
+    // do nothing, but necessary for the scala match
+  }
+
   Given(USING_DB) { (dbName: String) =>
     val builder = new GraphDatabaseFactory().newEmbeddedDatabaseBuilder(DatabaseLoader(dbName))
     graph = loadConfig(builder).newGraphDatabase()
   }
 
-  Given(INIT_DB) { (initQuery: String) =>
-    val builder = new TestGraphDatabaseFactory().newImpermanentDatabaseBuilder()
-    graph = loadConfig(builder).newGraphDatabase()
-    assert(!initQuery.contains("cypher"), "init query should do specify pre parser options")
-    // init update queries should go with rule+interpreted regardless the database configuration
-    graph.execute(s"cypher planner=rule runtime=interpreted $initQuery")
+  Given(ANY) {
+    initEmpty()
```

Comment:
We could do something fancy here, like randomising, just to make sure that the initial state does not affect the query result.